apps/events-helsinki/src/domain/search/eventSearch/constants.tsx

```diff
@@ -96,28 +96,28 @@ export enum EVENT_SEARCH_FILTERS {
 export const CATEGORY_CATALOG = {
   [EventTypeId.Course]: {
     default: [
-      COURSE_CATEGORIES.MOVIE,
-      COURSE_CATEGORIES.LANGUAGES,
-      COURSE_CATEGORIES.LITERATURE,
-      COURSE_CATEGORIES.ARTS_AND_CULTURE,
-      COURSE_CATEGORIES.VISUAL_ARTS,
-      COURSE_CATEGORIES.HANDICRAFTS,
-      COURSE_CATEGORIES.SPORT,
-      COURSE_CATEGORIES.MUSIC,
-      COURSE_CATEGORIES.GAMES,
-      COURSE_CATEGORIES.FOOD,
-      COURSE_CATEGORIES.DANCE,
-      COURSE_CATEGORIES.THEATRE,
+      EVENT_CATEGORIES.MOVIE,
+      EVENT_CATEGORIES.LANGUAGES,
+      EVENT_CATEGORIES.LITERATURE,
+      EVENT_CATEGORIES.ARTS_AND_CULTURE,
+      EVENT_CATEGORIES.VISUAL_ARTS,
+      EVENT_CATEGORIES.HANDICRAFTS,
+      EVENT_CATEGORIES.SPORT,
+      EVENT_CATEGORIES.MUSIC,
+      EVENT_CATEGORIES.GAMES,
+      EVENT_CATEGORIES.FOOD,
+      EVENT_CATEGORIES.DANCE,
+      EVENT_CATEGORIES.THEATRE,
     ],
     landingPage: [
-      COURSE_CATEGORIES.LITERATURE,
-      COURSE_CATEGORIES.VISUAL_ARTS,
-      COURSE_CATEGORIES.HANDICRAFTS,
-      COURSE_CATEGORIES.SPORT,
-      COURSE_CATEGORIES.MUSIC,
-      COURSE_CATEGORIES.GAMES,
-      COURSE_CATEGORIES.DANCE,
-      COURSE_CATEGORIES.THEATRE,
+      EVENT_CATEGORIES.LITERATURE,
+      EVENT_CATEGORIES.VISUAL_ARTS,
+      EVENT_CATEGORIES.HANDICRAFTS,
+      EVENT_CATEGORIES.SPORT,
+      EVENT_CATEGORIES.MUSIC,
+      EVENT_CATEGORIES.GAMES,
+      EVENT_CATEGORIES.DANCE,
+      EVENT_CATEGORIES.THEATRE,
     ],
   },
 };
```

Comment:
All these objects and arrays instroduced here are meant to be immutable so they should have const assertions set: 
https://blog.logrocket.com/const-assertions-are-the-killer-new-typescript-feature-b73451f35802/

Reply:
Added const assertions in 59621d4d93cd0319721de284638941dd00107bce